### ISSUE TITLE
[UI] Status bar eolFormat StatusLabel needs to be wider

### ISSUE THREAD

**Issue by inkydragon** [dail8859/NotepadNext#148]
![image](https://user-images.githubusercontent.com/5158738/171012470-9ec4ee8b-7365-4f13-ab56-090aff8db1b5.png)

https://github.com/dail8859/NotepadNext/blob/6888d4b9fb7544a5ba9ce06228ba603f37e06e5d/src/NotepadNext/widgets/EditorInfoStatusBar.cpp#L38

Maybe `120`?
```diff
-     eolFormat = new StatusLabel(100);
+     eolFormat = new StatusLabel(120);
```
![image](https://user-images.githubusercontent.com/5158738/171013800-588407f4-e8ab-4f37-9323-6ba3005ee1c2.png)

**Comment by inkydragon** [comment]
```cpp
    qInfo("rawName: %s", qUtf8Printable(app.font().rawName()));
    qInfo("family: %s", qUtf8Printable(app.font().family()));
    qInfo("defaultFamily: %s", qUtf8Printable(app.font().defaultFamily()));
    qInfo("styleName: %s", qUtf8Printable(app.font().styleName()));
    qInfo("toString: %s", qUtf8Printable(app.font().toString()));
```

```
[     0.174] I: rawName: unknown
[     0.174] I: family: SimSun
[     0.939] W: Unable to enumerate family ' "Ubuntu Mono derivative Powerline" '
[     1.057] I: defaultFamily: SimSun
[     1.057] I: styleName: 
[     1.057] I: toString: SimSun,9,-1,5,50,0,0,0,0,0
```

So, maybe `SimSun @ 10pt`

![image](https://user-images.githubusercontent.com/5158738/171253124-200cd08c-783e-49c6-9061-11fe5e9e4558.png)

I'am using Windows 10.

**Comment by dail8859** [comment]
Thanks for that additional info!

> I'am using Windows 10.

Interesting. Any idea why it is using a non-standard font? Did you change the default Font in Windows? If so this will give me a way to reproduce the issue.

**Comment by inkydragon** [comment]
> Any idea why it is using a non-standard font? Did you change the default Font in Windows?

I'm not sure. My system language setting is Simplified Chinese (zh-CN), may be related to this.
I guess you can simulate this by manually specifying QT's font settings: [QFont Class | Qt GUI 5.15.9](https://doc.qt.io/qt-5/qfont.html)

Some additional information: 
I'm not quite sure of the exact definition of the `default font`.
But [`noMeiryoUI`](https://github.com/Tatsu-syo/noMeiryoUI) tells me that my global default font is: `Microsoft YaHei UI 9pt`.

It looks like QT will read the system settings if the default font is not set.

- [Where does Qt get the default application font? | Qt Forum](https://forum.qt.io/topic/86872/where-does-qt-get-the-default-application-font/8)

**Comment by inkydragon** [comment]
Maybe this issue has been fixed since v0.6?
I'd like to close this.

![image](https://user-images.githubusercontent.com/5158738/214569285-2d78920d-a289-484d-a634-78cd2c3e7c14.png)

**Comment by dail8859** [comment]
> Maybe this issue has been fixed since v0.6?

Great to hear! Most likely the upgrade to Qt6 fixed the issue for us :)
